deploy/helm/elastic-agent/README.md

```diff
@@ -5,7 +5,7 @@
 
 # elastic-agent
 
-![Version: 9.0.0-beta](https://img.shields.io/badge/Version-9.0.0--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 9.0.0](https://img.shields.io/badge/AppVersion-9.0.0-informational?style=flat-square)
+![Version: 9.1.0-beta](https://img.shields.io/badge/Version-9.1.0--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 9.1.0](https://img.shields.io/badge/AppVersion-9.1.0-informational?style=flat-square)
```

Comment:
We should file an issue to automatically update versions in the README.

Reply:
that's a fair point! ty for raising it, created [this issue](https://github.com/elastic/elastic-agent/issues/7489) 🙂